ArkhamDisplay/BaseWindow.cs

```diff
@@ -380,6 +390,11 @@ protected virtual void UpdatePercent(int doneEntries, int totalEntries)
 
         riddleCounter.Text = GetRiddleCount();
 
+        if (IsFiltered)
+        {
+            riddleCounter.Text = $"{doneEntries}/{totalEntries}";
```

Comment:
This behaviour is incorrect. The riddle count and the number of entries are separate concepts.